rocky/reports/templates/partials/report_header.html

```diff
@@ -13,7 +13,12 @@ <h1>{{ report_name }}</h1>
                         <li>
                             <a href="{{ report_download_pdf_url }}"
                                target="_blank"
-                               rel="noopener noreferrer"><span class="icon ti-download" aria-hidden="true"></span>{% translate "Download report" %}</a>
+                               rel="noopener noreferrer"><span class="icon ti-download" aria-hidden="true"></span>{% translate "Download report (PDF)" %}</a>
+                        </li>
+                        <li>
+                            <a href="{{ report_download_json_url }}"
+                               target="_blank"
+                               rel="noopener noreferrer"><span class="icon ti-download" aria-hidden="true"></span>{% translate "Download report (JSON)" %}</a>
```

Comment:
This produces a duplicate `Download report` string to be translated. Both `(PDF)` and `(JSON)` strings don't have to be translated. 

E.g. `{% translate "Download report" %} (PDF)` instead

Reply:
Good one @ammar92